embassy-stm32/src/usart/buffered.rs

```diff
@@ -304,6 +332,10 @@ impl<'d, T: BasicInstance> BufferedUartRx<'d, T> {
             T::Interrupt::pend();
         }
     }
+
+    pub fn reconfigure(&mut self, config: &Config) {
```

Comment:
this should be named `set_config` for consistency. (the overlap between inherent and trait method is fine. Rust will silently prefer the inherent one, and since they have the same signature that will always work.)

Reply:
I was following from SPI: https://github.com/embassy-rs/embassy/blob/3ddc9cd110cb56f2ba9ac491b42df19a9b9c3fd4/embassy-stm32/src/spi/mod.rs#L1062-L1067. Would you like me to change that method to set_config too?

Reply:
oh oops. yes please :sweat_smile: